3-reveal/migrations/4-FI/1-Thailand-2019/deploy/events_jurisdictions_materialized_view.psql

```diff
@@ -0,0 +1,33 @@
+-- Deploy reveal_fi_thailand_2019:events_jurisdictions_materialized_view to pg
+-- requires: reveal_transaction_tables:events
+-- requires: reveal_transaction_tables:plans
+-- requires: reveal_transaction_tables:goals
+-- requires: reveal_transaction_tables:actions
+-- requires: utils:refresh_mat_views
+
+BEGIN;
+
+SET search_path TO :"schema",public;
+
+CREATE MATERIALIZED VIEW IF NOT EXISTS events_jurisdictions_materialized_view
+AS
+SELECT
+    events.id AS events_id,
+    events.details->'case_classification' AS case_classification,
+    events.details->'case_number' AS case_number,
+    events.details->'investigtion_date' AS date_of_diagnosis,
+    plans.identifier AS plan_id,
+    goals.goal_id,
+    events.location_id AS jurisdiction_id,
+    actions.code AS action_code
+FROM events
+LEFT JOIN plans
+    ON plans.fi_opensrpeventid = events.id
+LEFT JOIN goals
+    ON goals.plan_id = plans.identifier
+LEFT JOIN actions
+    ON actions.plan_id = plans.identifier
+WHERE events.event_type = 'Case Details'
+ORDER BY events.id;
```

Comment:
I think we can remove this ordering coz:

1. we don't need to produce ordered results
2. we pay a cost to order the query